libs/mock-builder/src/lib.rs

```diff
@@ -93,7 +289,7 @@ where
 }
 
 /// Register a mock function into the mock function storage.
-/// Same as `register()` but with using the locator who calls this macro.
+/// Same as `register()` but it uses as locator who calls this macro.
```

Comment:
Should this mean that it is use**d as** a locator or that it use**s a** locator?

Reply:
🤔
The macro generates a locator parameter that is passed to the `register()` function, That locator represents the position of the caller in the code. So I think "it uses as locator" is ok, but not sure if I get to transmit the idea correctly

Reply:
Thanks for clariyfing. Then let's leave it as is.